src/containers/AppBody.jsx

```diff
@@ -17,7 +18,8 @@ const styles = {
 class AppBody extends Component {
   componentDidMount() {
     const { dispatch, snippetKey } = this.props;
-    axios.get("../snippet.json")
+    const [user, snippet] = snippetKey.split("/");
```

Comment:
Now that I think about it, the key should be URI encoded so there _shouldn't_ be a '/', but I might be mistaken